### DISCUSSION TITLE
Scratch Logo Override

### DISCUSSION THREAD

**Discussion by GoldcatGames** [ScratchAddons/ScratchAddons#4467] [Ideas] [answered]
Would it be possible to upload a image to replace the scratch logo with that image, just to give it a little customization?

**Comment by WorldLanguages** [comment, marked as answer]
Probably rejected.
Try writing your own userstyle with Stylus?

**Reply by GoldcatGames** [reply]
I tried a lot of times, still can't figure it out, I would have to make a new class named a different thing, and I'm not that good at other types of code except C# and of course, scratch, here's the code to show the logo that I found via Right-Click Inspect:
`<li class="logo">`

**Reply by Chiroyce1** [reply]
This should work
```css
.logo > a {
    background-image: url("url-to-new-image") !important;
}
```

**Reply by GoldcatGames** [reply]
thanks
